src/test/java/bio/terra/workspace/service/resource/controlled/flight/CreateGcsBucketStepTest.java

```diff
@@ -49,9 +51,12 @@
 
   @Test
   public void testCreatesBucket() throws RetryException, InterruptedException {
+    final ApiGcpGcsBucketCreationParameters creationParameters =
```

Comment:
The tests in this file are still disableld. Fixed in https://github.com/DataBiosphere/terra-workspace-manager/pull/333